helix-view/src/editor.rs

```diff
@@ -246,6 +246,9 @@ pub enum StatusLineElement {
 
     /// The cursor position
     Position,
+
+    /// A single space
+    Spacer,
```

Comment:
`GutterType` uses `Padding` terminology.
`helix-view/src/editor.rs` on line 330.

Reply:
Gutters should also use the spacer terminology, I'll rename there